packages/ssz/src/util/strings.ts

```diff
@@ -0,0 +1,31 @@
+// Convert camelCase strings to various other formats  -- assumes input field is camelCase or camel1Case
+
+export const Case = {
+  snake: (field: string): string =>
+    field
+      .replaceAll(/[^0-z]/g, "")
+      .replaceAll(/[a-z][A-Z]|[0-9][A-Z]/g, (substr) => substr[0] + "_" + substr[1].toLowerCase()),
+  constant: (field: string): string =>
+    field
+      .replaceAll(/[^0-z]/g, "")
+      .replaceAll(/[a-z][A-Z]|[0-9][A-Z]/g, (substr) => substr[0] + "_" + substr[1])
+      .toUpperCase(),
+  pascal: (field: string): string => {
+    const first = field[0].toUpperCase();
+    return (first + field.slice(1)).replaceAll(/[^0-z]/g, "");
+  },
+  camel: (field: string): string => {
+    return field[0].toLowerCase() + field.slice(1);
+  },
+  header: (field: string): string => {
+    const first = field[0].toUpperCase();
+    return (
+      first +
+      field
+        .slice(1)
+        .replaceAll(/[^0-z]/g, "")
+        .replaceAll(/[a-z][A-Z]|[0-9][A-Z]/g, (substr) => substr[0] + "-" + substr[1])
+    );
+  },
+  eth2: (field: string): string => Case.snake(field),
```

Comment:
Can you incorporate https://github.com/ChainSafe/ssz/pull/296/files#diff-f75d38629191b6e32405d459655ce488f3d05f36e05e548a87fb3306f80ef72dR553-R555 here? so that can be simplified

Reply:
I think your deeplink might be broken but are you referring to [this line](https://github.com/ChainSafe/ssz/blob/b783bfbea4f456497950d8724937b098873cf2ef/packages/ssz/src/type/container.ts#L554)?  If so, just folded it in with my latest commit.